### DISCUSSION TITLE
seams to not work in packer config

### DISCUSSION THREAD

**Discussion by ModProg** [williamboman/nvim-lsp-installer#437] [Q&A] [answered]
I was using this, and expected it to install the servers, but it did not for some reason:
```lua
    use {
        'neovim/nvim-lspconfig',
        'williamboman/nvim-lsp-installer',
        config = function()
            lsp_installer = require "nvim-lsp-installer"

            local servers = {
                "bashls", "pyright", "vuels", "yamlls", "tailwindcss",
                "gdscript"
            }

            for _, name in pairs(servers) do
                local server_is_found, server = lsp_installer.get_server(name)
                if server_is_found and not server:is_installed() then
                    print("Installing " .. name)
                    server:install()
                end
            end
        end
    }
```

**Comment by ModProg** [comment]
I am not sure if this is an acutual issue or just a mistake by me, but it worked when I put the same code in the vimrc

**Reply by williamboman** [reply]
Hello! Just to make sure - have you run either `:PackerCompile` or `:PackerSync` after adding the `config` function?

**Reply by ModProg** [reply]
I thought I had tried the exact thing I posted above, but apparently I did not.

I had both lspconfig and lsp-installer in the same use call, and then config seams to not work.

I separated them now and it works now. 

I really thought I had tried that. 

Thanks anyway, in the end it was a mistake on my side.

**Comment by ModProg** [comment, marked as answer]
The problem is, config does not work when specifying multiple plugins in one use call.
